src/test/regress/pg_regress_multi.pl

```diff
@@ -71,6 +80,13 @@ ()
     $ENV{PATH} = "$libdir:".($ENV{PATH} || '');
 }
 
+my $plainRegress = "$pgxsdir/src/test/regress/pg_regress";
+my $isolationRegress = "${postgresBuilddir}/src/test/isolation/pg_isolation_regress";
+if ($isolationtester && ! -f "$isolationRegress")
+{
+    die "isolationtester not found at $isolationRegress\n";
```

Comment:
I hit this error because isolationtester needs to be explicitly built. Might be worth adding an instruction to the error message.